build/azure-pipelines/darwin/createDrop.sh

```diff
@@ -6,7 +6,9 @@ REPO="$(pwd)"
 mkdir -p $REPO/.build/darwin/{archive,server}
 
 # remove pkg from archive
-zip -d $REPO/.build/darwin/archive/azuredatastudio-darwin.zip "*.pkg"
+if [[ "signed" == "true" ]]; then
```

Comment:
I think you want `"$SIGNED" == "true"`

Reply:
Please also verify it actually just sign if you set signed = true in the build

Reply:
Whoops yep